src/styles/index.scss

```diff
@@ -2,2 +1,132 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+p,
+h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.body {
   margin: 0;
+  padding: 0;
+
+  --white: #fff;
+  --blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__info {
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    display: flex;
+  }
+
+  &__image {
+    margin: 0 auto 40px;
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+}
+
+.reviews {
+  min-width: 166px;
+  display: flex;
+  margin: 16px 0 24px;
+
+  &__text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    display: flex;
+    justify-content: space-between;
+    padding-left: 17px;
+  }
+}
+
+.stars {
+  min-width: 60%;
+  line-height: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 60%;
+    line-height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+  }
+
+  &--0 &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+  }
+}
+
+.price {
+  width: 166px;
+  line-height: 18px;
+  display: flex;
+  justify-content: space-between;
+  margin: 24px 0 16px;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    justify-content: space-between;
+  }
+
+  &__number {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+}
+
+.footer {
+  margin: 16px 0 0;
+  width: 166px;
+  height: 40px;
+  background: var(--blue);
+  border-radius: 5px;
+
+  &__button {
+    margin: 12px 70px;
```

Comment:
The margin for `.footer__button` is set to `12px 70px`, which might not center the button as expected. Ensure that the button is properly aligned within the footer, especially if the footer's width changes.